config/default_configs/default_config.yml

```diff
@@ -324,6 +324,9 @@ turbconv:
 advection_test:
   help: "Switches off all grid-scale and subgrid-scale momentum tendencies [`false` (default), `true`]"
   value: false
+edmfx_scale_blending:
```

Comment:
If we don't anticipate any more options, we could go for `edmfx_smooth_scale_blending` and keep simpler `true` vs `false` flags. I think this would also simplify the `EDMFXModel` `struct` in `solver/types.jl` because you wouldn't have to define another type for it.

I have no idea how much it matter for compilation times or run times.